tools/releng/launch-build

```diff
@@ -131,20 +140,16 @@ else
 	CURL_OPTS=(-s)
 fi
 
-CRUMB_URL="$JENKINS_URL/crumbIssuer/api/xml?xpath=concat(//crumbRequestField,\":\",//crumb)"
+if [[ -n "$BUILD_PARAM" ]]; then
+	JENKINS_JOB_PARAMS="--data-urlencode json={\"parameter\":[$BUILD_PARAM]}"
+else
+	JENKINS_JOB_PARAMS="--data-urlencode json={\"parameter\":[]}"
```

Comment:
Perhaps a comment here that we need to POST *some* data, even if it is effective empty, otherwise the /job/joyent-org/... build endpoint will error out?

Reply:
Good idea, I'll mention that.